test/e2e/__utils__/getLintStagedExecutor.js

```diff
@@ -0,0 +1,14 @@
+import { promisify } from 'node:util'
+import { resolve } from 'node:path'
+
+const exec = promisify(require('child_process').exec)
+const lintStagedPath = '../../../bin/lint-staged.js'
```

Comment:
Looks like the test is failing on Windows runners. Maybe using `execa` here would help?

Reply:
I saw some Linux runs failing but let's try it. Switched to execa.

Reply:
Maybe the diff argument needs to run differently on Github CI. I had to prefix the remote name (origin) to the branch name to make it work: https://github.com/elias-pap/youtube-share-clip/blob/main/.github/workflows/ci.yml#L25

Reply:
Just to be sure, `execa` can receive a `{ cwd }` as the second argument to make sure it's running in the correct directory.

You could also add the `--debug` flag if it might give more info.

Reply:
Added in 12c4218.

Reply:
I'm testing the CI on the fork. Will let you know.

Reply:
Tests pass now afaict.